roles/slapd/templates/bootstrap.ldif

```diff
@@ -12,4 +12,4 @@ olcPidFile: /var/run/openldap/openldap.pid
 dn: olcDatabase={0}config,cn=config
 objectClass: olcDatabaseConfig
 olcDatabase: {0}config
-olcRootPW:: {{ slapd_olcRootPW | b64encode }}
+olcRootPW:: {{ slapd_olcRootPW|b64encode }}
```

Comment:
jinja spec specifies `{{ foo|filter }}`.  Chromium does [weird things](https://www.chromium.org/developers/jinja#TOC-Spacing), but most jinja code online does not put spaces between pipes